portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -180,6 +180,39 @@ definition {
 		}
 	}
 
+	@description = "LRQA-55802, Validation of images for Sign In and Sign Out "
```

Comment:
Remove LRQA-55802
no trailing whitespace
Use: `This test validates sign in and sign out UI`